cli/proc_state.rs

```diff
@@ -340,15 +339,17 @@ impl ProcState {
     let analyzer = self.parsed_source_cache.as_analyzer();
 
     log::debug!("Creating module graph.");
-    let mut graph = self.graph_data.read().graph_inner_clone();
```

Comment:
I was not thinking about this situation when I wrote this code. Essentially we would take the graph out of the graph data, modify it, then put it back, but multiple graph loads might have been occurring here. A downside is this change makes each dynamic import load sequentially since deno_graph currently does not have a way for multiple loads to update it at the same time.

Reply:
I think that's fair in this situation

Reply:
Yeah, it's only non-concurrent in this section of the loading. The rest is still concurrent.